scala/lambdas/ingest-find-existing-asset/src/main/scala/uk/gov/nationalarchives/ingestfindexistingasset/Lambda.scala

```diff
@@ -64,9 +85,12 @@ class Lambda extends LambdaRunner[Input, StateOutput, Config, Dependencies] {
 object Lambda {
   case class Config(apiUrl: String, secretName: String, dynamoTableName: String) derives ConfigReader
 
-  case class Input(id: UUID, batchId: String)
+  case class InputItems(id: UUID, batchId: String)
+  case class OutputItems(id: UUID, batchId: String, assetExists: Boolean)
```

Comment:
1. Don't know. Got confused. I've changed it to Item
2. Ok, the lambda is called find-existing-asset.
3. Yes it would :smile: We don't really care where it exists, just that it does. If we end up moving off a dedicated preservation system onto to central bucket storage, or putting it in DRI :cry:, or printing the bits on A4 paper or anything, we want the code to still make sense.